flask_iiif/decorators.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Flask-IIIF
+# Copyright (C) 2015 CERN.
+#
+# Flask-IIIF is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Flask-IIIF decorators."""
+
+from functools import wraps
+
+from flask import current_app
+from flask_restful import abort
+from werkzeug import LocalProxy
+
+from .errors import (
+    IIIFValidatorError, MultimediaError, MultimediaImageForbidden,
+    MultimediaImageFormatError, MultimediaImageNotFound,
+    MultimediaImageQualityError, MultimediaImageRotateError,
+    MultmediaImageCropError, MultmediaImageResizeError
+)
+
+__all__ = ('api_decorator', 'error_handler', )
+
+_iiif = LocalProxy(lambda: current_app.extensions['iiif'])
```

Comment:
in order to have consistent naming I would go for `current_iiif` if you really need this proxy.